guides/15_Background_Filter_Video_Processor.md

```diff
@@ -1,46 +1,46 @@
+Background blur and replacement are integrated into the [browser demo](https://github.com/aws/amazon-chime-sdk-js/tree/master/demos/browser) application. To try it out, launch the demo with `npm run start`, join the meeting, click on the camera icon to enable video, then select the video filter drop down and select `Background Blur` or `Background Replacement`.
 
-## Background blur on the web
+## Background filter on the web
 
-Background blur runs in the end user’s web browser, leveraging modern web platform features including WebAssembly and SIMD. See the section “[Can I use background blur in my application?](#can-i-use-amazon-background-blur-in-my-application)” for browser version requirements.
+A background filter runs in the end user’s web browser, leveraging modern web platform features including WebAssembly and SIMD. See the section “[Can I use a background filter in my application?](#can-i-use-a-background-filter-in-my-application)” for browser version requirements.
 
-Background blur is available as part of the Amazon Chime SDK for JavaScript from version 2.20 onward.
+Background blur is available as part of the Amazon Chime SDK for JavaScript from version 2.20 onward. 
+Background replacement is available as part of the Amazon Chime SDK for JavaScript from version 2.24 onward.
 
-## Can I use background blur in my application?
+## Can I use a background filter in my application?
 
 ### Browser compatibility
 
-Background blur in the Amazon Chime SDK for JavaScript works in Firefox, Chrome, and Chromium-based browsers (including Electron) on desktop, Android, and Safari 14.1 and later on macOS. 
+A background filter in the Amazon Chime SDK for JavaScript works in Firefox, Chrome, and Chromium-based browsers (including Electron) on desktop, Android, and Safari 14.1 and later on macOS. 
 
 There is a known issue with `VideoFrameProcessor` in Safari 15: see [github issue 1059](https://github.com/aws/amazon-chime-sdk-js/issues/1059)
 
-Background blur can be CPU-intensive and the web runtime affects performance. As such, not all mobile devices or lower-spec laptop or desktop computers will be sufficiently powerful, or will be able to use background blur while also supporting multiple video streams and rich application functionality.
+A background filter can be CPU-intensive and the web runtime affects performance. As such, not all mobile devices or lower-spec laptop or desktop computers will be sufficiently powerful, or will be able to use a background filter while also supporting multiple video streams and rich application functionality.
 
-See the sections “[Checking for support before offering background blur](#checking-for-support-before-offering-background-blur)” for more details about checking for support.
+See the sections “[Checking for support before offering background filter](#checking-for-support-before-offering-background-filter)” for more details about checking for support.
```

Comment:
Missing `a`. 
- `checking-for-support-before-offering-background-filter` (this link)
- `checking-for-support-before-offering-a-background-filter` (an actual section title)